src/js/common/components/Widgets/ReadMore.jsx

```diff
@@ -27,6 +30,43 @@ export default class ReadMore extends Component {
   toggleLines (event) {
     event.preventDefault();
     const { readMore } = this.state;
+    const { location: { pathname: currentPathname } } = window;
+    const currentPage = lookupPageNameAndPageTypeDict(currentPathname);
+
+    const {
+      eventName = 'clickShowMoreAboutPolitician',
```

Comment:
Instead of 'eventName', please make this 'buttonId'. Let's not try to set a default value.